lib/chargify_api_ares/resources/subscription.rb

```diff
@@ -138,6 +138,22 @@ def delayed_cancel(cancel_at_end_of_period = true)
       save
     end
 
+    def product_change(handle)
+      self.product_handle = handle
+      save
+    end
+
+    def delayed_product_change(handle)
```

Comment:
Would you mind combining the `product_change` and `delayed_product_change` methods into just `product_change` with a default of `delayed = false`.

EG

```ruby
def product_change(handle, delayed = false)
  self.product_handle = handle
  self.product_change_delayed = delayed
end
```